lib/config/configuration.go

```diff
@@ -2783,15 +2783,16 @@ func isCmdLabelSpec(spec string) (types.CommandLabel, error) {
 // a given IP
 func applyListenIP(ip net.IP, cfg *servicecfg.Config) {
 	listeningAddresses := []*utils.NetAddr{
-		&cfg.Auth.ListenAddr,
```

Comment:
This duplicate line has been in the code since at least Teleport 7, but I suspect even longer!